src/components/dialogs/dialogs.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Dialog from 'react-toolbox/lib/dialog';
+import AppBar from 'react-toolbox/lib/app_bar';
+import { IconButton } from 'react-toolbox/lib/button';
+import Navigation from 'react-toolbox/lib/navigation';
+
+import VerifyMessage from '../signVerify/verifyMessage';
+import SignMessage from '../signVerify/signMessage';
+import styles from './dialogs.css';
+
+const Dialogs = props => (
+  <div>
+    {[
```

Comment:
The name of this file implies it's a generic component to handle any kind of child component. having configurations for *verify-message* and *sign-message* means we'll need the same configurations for all the components we may need later, yielding a long list of configurations. Please try writing a component that we can use it passing some properties.